pom.xml

```diff
@@ -3,31 +3,53 @@
     xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>net.mekomsolutions</groupId>
-    <artifactId>eip-routes-senaite</artifactId>
+    <groupId>com.ozonehis</groupId>
+    <artifactId>eip-openmrs-senaite-routes</artifactId>
```

Comment:
→ `eip-openmrs-senaite`

[This one](https://github.com/ozone-his/eip-keycloak-openmrs/blob/591df28266404359c153029ee686bfbce1840675/routes/pom.xml#L12) for Keycloak needs to be adapted as well to `eip-keycloak-openmrs`.

(The transient EIP Client there (pending the overall upgrade) will be named `eip-client-keycloak-openmrs`.)